src/shady-css/lexer.js

```diff
@@ -0,0 +1,141 @@
+import { matcher, tokenType } from './common';
+
+class Lexer {
```

Comment:
True, although "Tokenizer" always seemed kind of colloquial to me. Maybe that's fine, though, since this is JS.